### ISSUE TITLE
Accessibility violations in <Select> elements

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#26129]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

There are a11y violations in some of our <Select> implementations that need to be fixed. 

### Version

22.0.5

### Expected behavior

aria-label attributes are added to <Select> elements.

### Actual behavior

aria-label attributes are missing in the <Select> elements.

### How to Reproduce?

Going through the app where <Select> elements are implemented and scanning for a11y violations.
